app/assets/javascripts/components/reward_page_components/EditPersonLink.jsx

```diff
@@ -0,0 +1,33 @@
+class EditPersonLink extends React.Component {
+  constructor(props) {
+    super(props);
+    //this.timer = null;
+    this.timer = null;
+    this.add_emphasis = false;
+  }
+
+  decorate_link(url, text, person) {
+    if (!this.add_emphasis && person === undefined || person.iban !== null) {
```

Comment:
&& bindt toch sterker dan || in LTR operator precedence? Gaat person.iban hier niet fout als person undefined is? Zelfs als het goed gaat zou ik voor de duidelijkheid haakjes aanraden:

```js
 if (!this.add_emphasis && (person === undefined || person.iban !== null)) {
```